Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,14 @@
 ﻿using System;
+using System.IO;
```

Comment:
Codacy found an issue: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5768165)